x/twasm/keeper/genesis.go

```diff
@@ -10,17 +10,24 @@ import (
 	"github.com/confio/tgrade/x/twasm/types"
 )
 
+type noopValsetUpdater struct {
+}
+
+func (n noopValsetUpdater) ApplyAndReturnValidatorSetUpdates(context sdk.Context) (updates []abci.ValidatorUpdate, err error) {
```

Comment:
But don't we need this? Or at least for x/poe to be able to pass in.

How do the poe contracts set initial valset now?

Reply:
Ah, thank you. That makes sense